.circleci/config.yml

```diff
@@ -31,122 +46,91 @@ jobs:
                 PARALLEL: OFF
-
         steps:
             - checkout
-            - run:
-                name: apt-get update
-                command: sudo apt-get update
-            - run:
-                name: install dependencies
-                command: sudo apt-get install libgmp-dev libedit-dev bison flex cmake opam
-            - run:
-                name: install modern model validation environment
-                command: |
-                  opam init -y --disable-sandboxing # Note: do not disable sandboxing if running outside docker / other safety system
-                  opam update -y
-                  opam upgrade -y
-                  eval $(opam env)
-                  ./regression_models/bin/setup-env-modern.sh
-            - run:
-                name: Debug build gcc
-                command: ./ci/run_travis_commands.sh
-                environment:
-                    CMAKE_BUILD_TYPE: Debug
-                    ENABLE_LINE_EDITING: ON
-                    PARALLEL: ON
             - run:
                 name: Release build gcc
                 command: ./ci/run_travis_commands.sh
-                environment:
-                    CMAKE_BUILD_TYPE: Release
+
+    build-recent-clang-debug:
+        docker:
+          - image: verifyusi/verify-env:current
+            auth:
+                username: mydockerhub-user
+                password: $DOCKERHUB_PASSWORD
+            environment:
+                CMAKE_BUILD_TYPE: Debug
+                MODEL_VALIDATION: Dolmen
+                ENABLE_LINE_EDITING: ON
+                PARALLEL: ON
+                FLAGS: -Wall -Wextra -Werror -fsanitize=signed-integer-overflow,address,undefined
+                OSMT_INSTALL: ~/osmt-install
+        steps:
+            - checkout
             - run:
-                name: Install clang
-                command: |
-                    sudo apt-get install clang
-                    sudo update-alternatives --install /usr/bin/c++ c++ /usr/bin/clang++ 60
+                name: Setup clang
+                command: sudo update-alternatives --install /usr/bin/c++ c++ /usr/bin/clang++ 60
             - run:
                 name: Debug build llvm
-                command: ./ci/run_travis_commands.sh
-                environment:
-                    CMAKE_BUILD_TYPE: Debug
-                    FLAGS: -Wall -Wextra -Werror -fsanitize=signed-integer-overflow,address,undefined
+                command: |
+                    eval $(opam env)
+                    ./ci/run_travis_commands.sh
+
+    build-recent-clang-release:
+        docker:
+          - image: verifyusi/verify-env:current
+            auth:
+                username: mydockerhub-user
+                password: $DOCKERHUB_PASSWORD
+            environment:
+                CMAKE_BUILD_TYPE: Release
+                PARALLEL: ON
+                FLAGS: -Wall -Wextra -Werror
+                OSMT_INSTALL: ~/osmt-install
+        steps:
+            - checkout
```

Comment:
I think we are missing the `Setup clang` step which should be the same as in  `build-recent-clang-debug`, no?

Reply:
That's right.  Should be fixed now!